cli/src/main.rs

```diff
@@ -951,6 +951,27 @@ fn run<I: IntoIterator<Item = T>, T: Into<OsString> + Clone>(args: I) -> Result<
         )
     }
 
+    #[cfg(feature = "upgrade")]
+    {
+        app = app.subcommand(
+            SubCommand::with_name("upgrade")
+                .about("Upgrade splinter by importing YAML state files to a database")
+                .arg(
+                    Arg::with_name("state_dir")
+                        .value_name("state-dir")
+                        .short("S")
```

Comment:
Can you also add `.long("state-dir")` here and `.long("connect")` below

Reply:
Done